docs/sources/setup-grafana/configure-security/configure-authentication/azuread/index.md

```diff
@@ -65,19 +65,57 @@ To enable the Azure AD/Entra ID OAuth, register your application with Entra ID.
-        Add a new entry under Federated credentials with the following configuration.
+     - **_Managed Identity_**
 
-        - Federated credential scenario: Select **Other issuer**.
-        - Issuer: The OAuth 2.0 / OIDC issuer URL of the Microsoft Entra ID authority. For example: `https://login.microsoftonline.com/{tenantID}/v2.0`.
-        - Subject identifier: The Object (Principal) ID GUID of the Managed Identity.
-        - Name: A unique descriptive name for the credential.
-        - Description: Grafana OAuth.
-        - Audience: The audience value that must appear in the external token. For Public cloud, it would be `api://AzureADTokenExchange`. See mentioned documentation for the full list of available audiences.
+       1. Refer to [Configure an application to trust a managed identity (preview)](https://learn.microsoft.com/en-us/entra/workload-id/workload-identity-federation-config-app-trust-managed-identity?tabs=microsoft-entra-admin-center) for a complete guide on setting up a managed identity as a federated credential.
+          Add a new entry under Federated credentials with the following configuration.
 
-     1. Click **Add**, and then copy the Managed Identity Client ID and the federated credential Audience values. This is your OAuth 2.0 federated credential.
+          - Federated credential scenario: Select **Other issuer**.
+          - Issuer: The OAuth 2.0 / OIDC issuer URL of the Microsoft Entra ID authority. For example: `https://login.microsoftonline.com/{tenantID}/v2.0`.
+          - Subject identifier: The Object (Principal) ID GUID of the Managed Identity.
+          - Name: A unique descriptive name for the credential.
+          - Description: Grafana OAuth.
+          - Audience: The audience value that must appear in the external token. For Public cloud, it would be `api://AzureADTokenExchange`. See mentioned documentation for the full list of available audiences.
 
-     1. You must have set `client_authentication` under `[auth.azuread]` to `managed_identity` in the Grafana server configuration for this to work.
+       1. Click **Add**, and then copy the Managed Identity Client ID and the federated credential Audience values. This is your OAuth 2.0 federated credential.
+
+       1. You must have set `client_authentication` under `[auth.azuread]` to `managed_identity` in the Grafana server configuration for this to work.
+
+       1. You may optionally set `workload_identity_token_file` (env var `GF_AUTH_AZUREAD_WORKLOAD_IDENTITY_TOKEN_FILE`) under `[auth.azuread]` to `/var/run/secrets/azure/tokens/azure-identity-token` in the Grafana server configuration. This setting defaults to `/var/run/secrets/azure/tokens/azure-identity-token`.
```

Comment:
I think this is not needed here, because the workload identity token file is only used when the `client_authentication` is set to `workload_identity`. Could you please remove it?